Python/libraries/recognizers-text/recognizers_text/Matcher/ac_automaton.py

```diff
@@ -11,14 +17,64 @@ def __init__(self):
     def root(self) -> AaNode():
         return self.__root
 
-    # This class should be overrided from AbstractMatcher
     def insert(self, value: [], _id: str) -> None:
-        pass
+        node = self.root
+        i = 0
+
+        for item in value:
+            child = node[item]
+
+            if child is None:
+                node[item] = AaNode(item, i, node)
+                child = node[item]
+
+            node = child
+            i += 1
+
+        node.add_value(_id)
 
-    # This class should be overrided from AbstractMatcher
     def init(self, values: [], ids: []) -> None:
-        pass
+        self.batch_insert(values, ids)
+        _queue = Queue()
```

Comment:
The '_' means that 'queue' is for internal use? I am not sure as in the C# implementation it is not declared as a private var.